src/containers/SideNav/SideNav.js

```diff
@@ -214,6 +214,13 @@ export class SideNav extends Component {
           <SideNavMenuItem element={NavLink} icon={<span />} to="/secrets">
             Secrets
           </SideNavMenuItem>
+          <SideNavMenuItem
+            element={NavLink}
+            icon={<span />}
+            to="/serviceAccounts"
```

Comment:
use router utils when generating links, i.e. `urls.serviceAccounts.all()` instead of hardcoding `/serviceaccounts` here